### DISCUSSION TITLE
Proposal: Remove Default/Simple Theme

### DISCUSSION THREAD

**Discussion by robloo** [AvaloniaUI/Avalonia#7438] [General]
I'm a little biased as I sit here and write a default theme for a new control. However, I wonder why is this default theme still supported? Default is:

 - Inferior to Fluent v1 in terms of quality
 - Far inferior to Fluent v1 in terms of design fidelity and user experience
 - An unnecessary maintenance burden
 - Has clear errors (Foreground Brush doesn't change when checked):
    ![image](https://user-images.githubusercontent.com/17993847/151107247-968a46d2-5bad-4fbc-a6ae-a3b942b0c766.png)

About the only thing going for it is it's lightweight and what is used in legacy apps. However, Fluent v1 has been around for a few years now. It might be time to cut off the dead weight. Then Fluent v2 can be added in addition to -- or in replacement of -- Fluent v1.

Apps that need the old theme can either:
 - Keep using old versions of Avalonia
 - Copy the themes into their own project

I see Default theme as a lot of dead weight and wonder when is a good time to cut it loose.

**Comment by maxkatz6** [comment]
In next major release it will be moved out of main package and distributed as separated nuget package (same for Fluent).
We are not planning to completely remove it anytime soon. Although with stable API it could be used from nuget package even being out-of-dated.

**Comment by grokys** [comment]
Default/simple theme is still used in unit tests and by some of our customers running on low power devices, so it won't be removed. As @maxkatz6 says, it will be moved to a separate package for 11.0 though.

**Reply by robloo** [reply]
I'm surprised there are customers using this in production:

![image](https://user-images.githubusercontent.com/17993847/151107247-968a46d2-5bad-4fbc-a6ae-a3b942b0c766.png). 

Contrast and usability requirements are usually mandatory before shipping. I have to think they modified the style or never use a toggle button then. If they modified the style they certainly can pull the default styles into their own app.

Anyway, I understand if you have paying customers it is prudent to maintain perfect compatibility; I'm just surprised. Unit tests are a problem too.

I just hope you have have an idea how to sunset this in the future.

**Reply by maxkatz6** [reply]
Default one can be used as a base theme, on top of which they build their own themes. 
And yes, there are still applications that use default without any major changes as well. Been there.
